src/test/java/com/vmtapp/enterprise/VarmentTmsApplicationTests.java

```diff
@@ -151,7 +151,7 @@ private void whenPostTicketWithAttributes(){
         ticket.setEmail("johnsmith@company.com");
     }
 
-    private void thenReturnSavedTicketWithAttributes() {
+    private void thenReturnSavedTicketWithAttributes() throws Exception{
```

Comment:
Added Throw Exception to be more specific with Exception.
Source: https://stackify.com/best-practices-exceptions-java/